_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1108,12 +1149,6 @@ inputs = {
     "iam:GetRole",
     "iam:GetRolePolicy",
   ]
-
-  # Configures the auto deploy max session duration to be 4 hours.
```

Comment:
Why remove this code?

Reply:
CIS 1.3 suggests not to have any roles and permissions with deployment, or full-access, or dev-access. The only roles that should be configured are ssh, readonly, billing & support. (See CIS 1.3 1.15-1.18)

Reply:
Ah, I see. I got this from the Ref Arch generated files

Reply:
I double checked this now and the default value for `max_session_duration_human_users` is 43200 (12 hours) and for `max_session_duration_machine_users` is 3600 (1 hour)

Reply:
yep 👍 I am not sure if you want me to add these back?